syntax = "proto3";

message PaymentMessage {
    string order_id = 1;
    int32 user_id = 2;
    string payment_status = 3;
}